lib/elixir/test/elixir/module/types/expr_test.exs

```diff
@@ -193,12 +201,95 @@ defmodule Module.Types.ExprTest do
       assert {:error, _} = quoted_expr([a], case(a, do: (:foo = b -> :bar = b)))
 
       assert quoted_expr([a], case(a, do: (:foo = b -> :foo = b))) == {:ok, :dynamic}
+
+      assert quoted_expr(
```

Comment:
We should probably consider breaking these large tests into `describe "case"` and have a bit more describe test names. This will certainly only grow in example size. :)